RedditSharp/Reddit.cs

```diff
@@ -110,12 +121,17 @@ public Reddit(DefaultWebAgent.RateLimitMode limitMode, bool useSsl = true)
         /// <param name="username"></param>
         /// <param name="password"></param>
         /// <param name="useSsl"></param>
+        [Obsolete]
```

Comment:
Technically OAuth is supposed to be the only available option because the Reddit API says so, but since password/username authentication is done via normal browsers it will always be available, so it's deprecated, as it's available (and always will be), but it's not reccomended